.nvmrc

```
@@ -0,0 +1 @@
+12.0.0
```

We don't want to force it to use exactly version `12.0.0`. Setting this to simply `12` would have it use any version 12.

```suggestion
12
```

We may want to just use `lts/*` which has it use the latest LTS version.
```suggestion
lts/*
```